package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "classnames": "^2.2.6"
```

Comment:
we should not leak any of the UI files into the root kelp directory. do we need this file here or can it go inside `/gui/web`?